recipes/gtest/all/conanfile.py

```diff
@@ -78,6 +107,9 @@ def package_info(self):
         self.cpp_info.components["libgtest"].libs = ["gtest{}".format(self._postfix)]
         if self.settings.os == "Linux":
             self.cpp_info.components["libgtest"].system_libs.append("pthread")
+        
+        if self.settings.os == "Neutrino" and self.settings.os.version == "7.1":
+            self.cpp_info.components["libgtest"].system_libs.append("regex")
```

Comment:
Only this exact version requires a system lib?

Reply:
Since 7.1 regex was put in a separate library. Maybe also for next versions but there was over 3 years between 7.0 and 7.1 so I just put it at this one as there is no telling...